authorization/invitation/service/invitation_service.go

```diff
@@ -162,3 +211,98 @@ func (s *invitationServiceImpl) Issue(ctx context.Context, issuingUserId uuid.UU
 
 	return nil
 }
+
+type invitationNotification struct {
+	invitation *invitationrepo.Invitation
+	roles      []string
+}
+
+// processTeamInviteNotifications sends an e-mail notification to a user.
+func (s *invitationServiceImpl) processTeamInviteNotifications(ctx context.Context, team *account.Identity, inviterName string,
+	notifications []invitationNotification, witURL string) error {
+	teamName := team.IdentityResource.Name
+
+	var spaceName string
+	var err error
+
+	if witURL != "" {
+		spaceName, err = lookupSpaceName(ctx, witURL, *team.IdentityResource.ParentResourceID)
+		if err != nil {
+			return err
+		}
+	}
+
+	var messages []notification.Message
+
+	for _, n := range notifications {
+		acceptURL := fmt.Sprintf("%s/api/invitations/accept?code=%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
```

Comment:
It should actually use accept invitation client path instead of hardcoded string. But the client does not exist yet.
It's important to switch to the goa generated client path as soon as we have the accept invitation controller implemented.
Then it will work even if we update the endpoint path in our design doc later.

Reply:
I already had the same thought, and I'll roll this change into the accept invitation PR.